source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -1314,17 +1384,63 @@ bool ServerContextImpl::isClientEcdsaCapable(const SSL_CLIENT_HELLO* ssl_client_
   return false;
 }
 
+OcspStapleAction ServerContextImpl::passesOcspPolicy(const ContextImpl::TlsContext& ctx) {
```

Comment:
the function name and return type don't seem to match. Maybe name function `ocspPolicyAction` or something.